### PR TITLE
input text fields fixed in seed and send

### PR DESCRIPTION
**Pull request by juanky201271**
In some circumstances when you jump from an amount field to another, fails with an ugly NaN value. I fixed that and I upload a new version to both stores.
&
In small screens you can't fill the seed and birthday because of the lack of space. When the Keyboard are open I move the screen to up, the same like in the Send screen.

### CODE REVIEW

**Comment by zancas** [comment]
Can we separate concerns here?   I don't understand the `NaN` issue, and would like to before we merge a fix.

On the other hand the resize to allow visibility when the keyboard comes up issue that a User reported, can be merged right away.

File: components/Send/Send.tsx [hunk 1]
```diff
@@ -575,9 +578,7 @@ const Send: React.FunctionComponent<SendProps> = ({
                         }}
                         value={ta.amount.toString()}
                         onChangeText={(text: string) => updateToField(null, text.substring(0, 20), null, null)}
-                        onEndEditing={(e: any) =>
-                          updateToField(null, Number(e.nativeEvent.text.substring(0, 20)).toFixed(8), null, null)
-                        }
+                        onEndEditing={(e: any) => updateToField(null, e.nativeEvent.text.substring(0, 20), null, null)}
```
Comment (juanky201271) [inline components/Send/Send.tsx:582]: @zancas The problem here with NaN was: onEndEdiiting had a Number cast which is a problem when the value is not a number, like 0...0 for example... and in this case the value became in NaN. I had to remove Number here.

**Comment by juanky201271** [comment]
@zancas this is ready for review... please.

File: components/Seed/Seed.tsx [hunk 2]
```diff
@@ -217,7 +253,7 @@ const Seed: React.FunctionComponent<SeedProps> = ({ onClickOK, onClickCancel, ac
                     marginLeft: 5,
                   }}
                   value={birthdayNumber.toString()}
-                  onChangeText={(text: string) => setBirthdayNumber(Number(text))}
+                  onChangeText={(text: string) => setBirthdayNumber(text)}
```
Comment (zancas) [inline components/Seed/Seed.tsx:256]: So the cast moves?

File: components/Seed/Seed.tsx [hunk 3]
```diff
@@ -58,9 +62,33 @@ const Seed: React.FunctionComponent<SeedProps> = ({ onClickOK, onClickCancel, ac
     );
     setTimes(action === 'change' || action === 'backup' || action === 'server' ? 1 : 0);
     setSeedPhrase(walletSeed?.seed || '');
-    setBirthdayNumber(walletSeed?.birthday || 0);
+    setBirthdayNumber(walletSeed?.birthday?.toString() || '');
```
Comment (zancas) [inline components/Seed/Seed.tsx:65]: What happens when `Number('')` receives an empty string?

File: components/Seed/Seed.tsx [hunk 4]
```diff
@@ -46,10 +47,13 @@ const Seed: React.FunctionComponent<SeedProps> = ({ onClickOK, onClickCancel, ac
 
   const { colors } = useTheme() as unknown as ThemeType;
   const [seedPhrase, setSeedPhrase] = useState('');
-  const [birthdayNumber, setBirthdayNumber] = useState(0);
+  const [birthdayNumber, setBirthdayNumber] = useState('');
```
Comment (zancas) [inline components/Seed/Seed.tsx:50]: Is this the definition of `setBirthdayNumber`?

File: components/Seed/Seed.tsx (hunk 3 above)
Comment (juanky201271) [reply components/Seed/Seed.tsx:65]: The value in TextInput is always a string and if this value are empty, the field shows the placeholder...

**Review by zancas — APPROVED** [review]

**Pull request merged**